judge/models/tests/test_problem.py

```diff
@@ -156,6 +172,62 @@ def test_organization_private_problem_methods(self):
         }
         self._test_object_methods_with_users(self.organization_private_problem, data)
 
+    def test_organization_admin_private_problem_methods(self):
+        data = {
+            'staff_problem_edit_own': {
+                'is_accessible_by': self.assertFalse,
+                'is_editable_by': self.assertFalse,
+                'is_subs_manageable_by': self.assertFalse,
+            },
+            'staff_problem_see_all': {
+                'is_accessible_by': self.assertTrue,
+                'is_editable_by': self.assertFalse,
+                'is_subs_manageable_by': self.assertFalse,
+            },
+            'staff_problem_edit_all': {
+                'is_accessible_by': self.assertTrue,
+                'is_editable_by': self.assertTrue,
+            },
+            'staff_problem_edit_public': {
+                'is_accessible_by': self.assertTrue,
+                'is_editable_by': self.assertTrue,
+            },
+            'staff_problem_see_organization': {
+                'is_accessible_by': self.assertFalse,
+                'is_editable_by': self.assertFalse,
+            },
+            'normal': {
+                'is_accessible_by': self.assertFalse,
+                'is_editable_by': self.assertFalse,
+            },
+            'anonymous': {
+                'is_accessible_by': self.assertFalse,
+                'is_editable_by': self.assertFalse,
+            },
+        }
+        self._test_object_methods_with_users(self.organization_admin_private_problem, data)
+
+    def test_organization_admin_problem_methods(self):
```

Comment:
Shouldn't you also test an organization admin on this problem?

Reply:
The `problem_organization` organization has `normal` and `staff_problem_edit_public` as organization admins (unless you mean a different organization admin (?)).

Reply:
Ah oops I didn't read that correctly.

Trying this with the admin of another organization would make for a good unit test.

Reply:
Done